lib/ts/recipe/session/recipeImplementation.ts

```diff
@@ -278,7 +298,8 @@ export default function getRecipeInterface(
                     response.session.userId,
                     response.session.userDataInJWT,
                     res,
-                    req
+                    req,
+                    transferMethod
```

Comment:
there is a clearSession in the catch below - should that also take in the transferMethod? In fact, why not make clearSession always taken in a transferMethod? This way, errors like this can be avoided.